assets/model_monitoring/components/src/generation_safety_quality/annotation_compute_histogram/run.py

```diff
@@ -1000,9 +1000,11 @@ def __init__(self, n_retry, backoff_factor):
 
         retry_strategy = Retry(
             total=n_retry,
+            status=n_retry,
             status_forcelist=[104, 408, 409, 424, 429, 500, 502, 503, 504],
             backoff_factor=backoff_factor,
             respect_retry_after_header=True,
+            allowed_methods=frozenset(['GET', 'POST', 'PUT']),
```

Comment:
this was the main issue, there was no retry because default allowed methods don't include POST which is what we need.